tests/BenchmarkDotNet.Tests/Exporters/MarkdownExporterVerifyTests.cs

```diff
@@ -40,9 +41,9 @@ public Task GroupExporterTest(Type benchmarkType)
 
             var logger = new AccumulationLogger();
             logger.WriteLine("=== " + benchmarkType.Name + " ===");
-
+            bool hasHideColumnsAttribute = benchmarkType.IsDefined(typeof(HideColumnsAttribute), false);
             var exporter = MarkdownExporter.Mock;
-            var summary = MockFactory.CreateSummary(benchmarkType);
+            var summary = hasHideColumnsAttribute ? MockFactory.CreateSummary(benchmarkType, new ColumnHidingByNameRule("StdDev")) : MockFactory.CreateSummary(benchmarkType);
```

Comment:
This can be done more generically. Add `using BenchmarkDotNet.Extensions;` and change it to this:

```suggestion
            var summary = MockFactory.CreateSummary(benchmarkType, benchmarkType.ResolveAttribute<HideColumnsAttribute>()?.Config.GetColumnHidingRules().ToArray() ?? []);
```

Reply:
Can we use GetCustomAttribute as Type does not contain definition for ResolveAttribute

Reply:
It's an extension. You need to add the using statement that I mentioned.